pkgs/by-name/cb/cbconvert/package.nix

```
@@ -0,0 +1,89 @@
+{
+  buildGoModule,
+  bzip2,
+  callPackage,
+  cbconvert,
+  fetchFromGitHub,
+  fetchpatch2,
+  imagemagick,
+  lib,
+  libunarr,
+  mupdf-headless,
+  nix-update-script,
+  pkg-config,
+  stdenv,
+  testers,
+  zlib,
+}:
+
+buildGoModule rec {
+  pname = "cbconvert";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "gen2brain";
+    repo = "cbconvert";
+    rev = "v${version}";
+    hash = "sha256-9x7RXyiQoV2nIVFnG1XHcYfTQiMZ88Ck7uuY7NLK8CA=";
+  };
+
+  # Update dependencies in order to use the extlib tag.
+  patches = [
+    (fetchpatch2 {
+      name = "update-dependencies-1.patch";
+      url = "https://github.com/gen2brain/cbconvert/commit/1a36ec17b2c012f278492d60d469b8e8457a6110.patch";
+      hash = "sha256-E+HWYPz9FtU3JAktzIRflF/pHdLfoaciBmjb7UOQYLo=";
+    })
+    (fetchpatch2 {
+      name = "update-dependencies-2.patch";
+      url = "https://github.com/gen2brain/cbconvert/commit/74c5de699413e95133f97666b64a1866f88fedd5.patch";
+      hash = "sha256-rrJsYJHcfNWF90vwUAT3J/gqg22e1gk6I48LsTrYbmU=";
+    })
+  ];
+
+  vendorHash = "sha256-aVInsWvygNH+/h7uQs4hAPOO2gsSkBx+tI+TK77M/hg=";
+  modRoot = "cmd/${pname}";
+
+  proxyVendor = true;
+
+  # The extlib tag forces the github.com/gen2brain/go-unarr module to use external libraries instead of bundled ones.
+  tags = [ "extlib" ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.appVersion=${version}"
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    bzip2
+    imagemagick
+    libunarr
+    mupdf-headless
+    zlib
+  ];
+
+  passthru = {
+    gui = callPackage ./gui.nix { };
+    updateScript = nix-update-script { };
+    tests.version = testers.testVersion {
+      package = cbconvert;
+      command = "cbconvert version";
+    };
+  };
+
+  meta = {
+    description = "A Comic Book converter";
+    homepage = "https://github.com/gen2brain/cbconvert";
+    changelog = "https://github.com/gen2brain/cbconvert/releases/tag/v${version}";
+    license = with lib.licenses; [ gpl3Only ];
+    platforms = with lib.platforms; linux ++ darwin ++ windows;
+    broken = !stdenv.hostPlatform.isLinux;
```

```suggestion
    platforms = lib.platforms.linux;
```
If it doesn't work, we should not set platforms for it.

---

@SuperSandro2000 I'll do that. It technically should work on Windows and macOS as well, but I haven't tested it there. I did this according to the recommendation [here](https://github.com/NixOS/nixpkgs/pull/346178#discussion_r1786394904) previously.

---

Done.

---

I have never tested any of my packages on darwin and some of them still have platforms.unix but I never ever test them on darwin because I just do not have any device from Apple.